unified-codes/apps/web/src/components/containers/detail/DetailEntityListItem.tsx

```diff
@@ -104,6 +104,7 @@ const DetailEntityListItemComponent: DetailEntityListItem = ({
           parentDescription = `${entityDescription} ${description}`;
           break;
         case 'unit_of_use':
+        case 'strength':
```

Comment:
Previously the lowest level was only 'unit_of_use' but now we can go down to strength, was causing entity descriptions to appear truncated

Reply:
Component no longer does specific type mapping so this change has also gone after merging in `master`